### PR TITLE
Fixed the welcome screen being blank after rotation 

### PR DESCRIPTION
**Pull request by Hashik-Donthineni**
Fixed the welcome screen, when you open the app for the first time, is showing a blank screen when you rotate the device (change orientation). I test this on API 18, 28.  There are no issues nor crashes. I just saved the instances and restored them.

This fixes the issue #304 
Before: 
Before
After:
After

### CODE REVIEW

File: app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java [hunk 1]
```diff
@@ -19,6 +22,8 @@
     private String mButtonText;
     private String mSubTitle;
     private View.OnClickListener mButtonListener;
+    private TextView tv, title;
```
Comment (bitmold) [inline app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java]: Perhaps `tv` should be renamed to something more specific

File: app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java [hunk 2]
```diff
@@ -2,62 +2,83 @@
 
 import android.content.Context;
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.os.Bundle;
+
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+
+import android.util.Log;
```
Comment (bitmold) [inline app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java]: unused import

File: app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java [hunk 3]
```diff
@@ -2,62 +2,83 @@
...
+
 import org.torproject.android.R;
+import org.torproject.android.service.util.Prefs;
 import org.torproject.android.settings.LocaleHelper;
 import org.torproject.android.ui.AppManagerActivity;
 import org.torproject.android.ui.VPNEnableActivity;
 import org.torproject.android.ui.hiddenservices.permissions.PermissionManager;
 
+import java.util.HashMap;
+import java.util.LinkedList;
```
Comment (bitmold) [inline app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java]: unused

File: app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java [hunk 4]
```diff
@@ -2,62 +2,83 @@
...
 import com.github.paolorotolo.appintro.AppIntro;
+
 import org.torproject.android.R;
+import org.torproject.android.service.util.Prefs;
 import org.torproject.android.settings.LocaleHelper;
 import org.torproject.android.ui.AppManagerActivity;
 import org.torproject.android.ui.VPNEnableActivity;
 import org.torproject.android.ui.hiddenservices.permissions.PermissionManager;
 
+import java.util.HashMap;
```
Comment (bitmold) [inline app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java]: unused

File: app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java [hunk 5]
```diff
@@ -57,24 +62,48 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
...
+            }
+
+        }
+    }
+
+    //Saving the data
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+        outState.putString(getResources().getString(R.string.Title), mTitle);
```
Comment (bitmold) [inline app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java]: I would change the names of these resource keys to pref_title or something so people don't change the value of the key and create duplicate preferences

File: app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java (hunk 1 above)
Comment (Hashik-Donthineni) [reply app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java]: I let the default names stay. Yeah, I should probably.

File: app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java (hunk 3 above)
Comment (Hashik-Donthineni) [reply app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java]: Oh, yeah. I forgot to remove it at the end.

File: app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java (hunk 5 above)
Comment (Hashik-Donthineni) [reply app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java]: Okay, will do.

File: app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java (hunk 3 above)
Comment (bitmold) [reply app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java]: cool, sorry about how terse my code review comments were earlier ~ fixing UX bugs in anonymity software is really important, this was a good find + fix :+)

Comment (Hashik-Donthineni) [reply app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java]: no issue :)

**Pull request merged**